cc/toolchains/llvm/cc_toolchain_config.bzl

```
@@ -35,7 +35,7 @@ def cc_toolchain_config(
     if not is_target_triplet(target_system_name):
         fail(target_system_name + " is not a target tripplet")
 
-    cross_compile = host_system_name != target_system_name
+    use_bundled_libcpp = builtin_sysroot != None
```

This won't quite work because the mac toolchains use the `builtin_sysroot` argument: https://github.com/swift-nav/rules_swiftnav/blob/af0ddfab4a282d642f00d4665aa31e342b4eaef3/cc/toolchains/llvm/aarch64-darwin/BUILD.bazel#L96 (and we still want to use libc++ in that context).

Maybe:
```suggestion
    use_bundled_libcpp = builtin_sysroot != None && !is_darwin
```